### ISSUE TITLE
css file imported as string?

### ISSUE THREAD

**Issue by Lisanna** [Quramy/typed-css-modules#57]
I install and run the module as expected, indeed the .d.ts file is successfully created.

My sample project structure is the following: 
![ps](https://user-images.githubusercontent.com/8490781/50692135-303a1480-1033-11e9-9af4-90c2a288afe1.png)

In the src folder I have 
![src](https://user-images.githubusercontent.com/8490781/50692188-58297800-1033-11e9-9a61-5d9ff7a29b88.png)

In feature-viewer.ts I import
`import * as teststyles from './assets/testcss.css';`

testcss.css (as in example)
`/* styles.css */
@value primary: red;
.myClass {
    color: primary;
}`

The .d.ts files in assets are generated by typed-css-modules execution and indeed the IDE is now able to access the css properties e.g. testcss.myClass.
![schermata](https://user-images.githubusercontent.com/8490781/50692722-16013600-1035-11e9-850f-9c5cfb1208bd.png)

However the styling still does not work because all values are undefined. 
Indeed the value of teststyles.myClass, if printed, is undefined.
Instead, teststyles, if printed, is 

> /* styles.css */
@value primary: red;
.myClass {
    color: primary;
}

Basically a string of the file content. Shouldn't it be some kind of object?
